libs/librepcb/projecteditor/schematiceditor/schematiceditor.ui

```diff
@@ -313,6 +319,9 @@
    <property name="text">
     <string>PDF Export</string>
    </property>
+   <property name="visible">
+    <bool>true</bool>
+   </property>
```

Comment:
Just a note: When modifying widget properties with QtCreator, it leads to changes in *.ui files even if the property was changed back to the default value (like here, `visible=true` is actually the default). To avoid cluttering up commits with such useless changes, this can be reverted by clicking on the button at the right of the property field:
![auswahl_014](https://user-images.githubusercontent.com/5374821/35415823-47447cda-0227-11e8-9c7f-60bf7838bd59.png)